### DISCUSSION TITLE
Inputs to software

### DISCUSSION THREAD

**Discussion by impredicative** [SakanaAI/AI-Scientist#8] [General]
Thank you for the software. At a high level, I want to know what are the inputs for a job. For a human to do science, these are needed:

1. Initial experimental data.
2. Past literature.
3. A loop driving experiments and collecting data.

When using AI-Scientist, are 1 and 2 above provided as inputs to the software? Or what does it require and use? If anything is provided to the software, in what formats can it be provided and how?

**Comment by conglu1997** [comment]
I would recommend reading the code, starting from `launch_scientist.py`

**Reply by BradKML** [reply]
Instead of reading the code, it is better to package this in a way that requires less direct file editing (or have all the files that customizes experiments to be located in the same folder). Probs warrant Pull request to twist the file structure a little for maximum usability.

**Comment by impredicative** [comment]
Cluttered.

**Reply by BradKML** [reply]
It needs a rewrite using an AI editor ngl, but other than that it is somewhat clear already.
